src/libs/Ideogram/openapi.yaml

```
@@ -1779,14 +1779,14 @@ components:
           - a cat
           - a dog
         sampling_request_params:
-          aspect_ratio: ASPECT_10_16
-          style_type: REALISTIC
+          aspect_ratio: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)